docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -1,6 +1,216 @@
 ---
-title: Standardize access to observability data
-description: Learn how the Keptn Lifecycle Toolkit standardizes access to observability data.
-weight: 45
+title: Orchestrating pre- and post-deployment tasks and evaluations
+description: Learn how the Keptn Lifecycle Toolkit can orchestrate deployment checks.
+weight: 55
 ---
 
+In this exercise, we will configure the Keptn Lifecyle Toolkit
+to run deployment checks as part of your deployment.
+Whether you are deploying your software with
+Argo, Flux, another deployment engine, or even `kubectl apply`,
+the Lifecycle Toolkit can do the following:
+
+* Pre-deploy: Validate external dependencies,
+  confirm that images are scanned, and so forth
+
+* Post-deply: Execute tests, notify stakeholders,
+  promote to the next stage
+
+* Automatically validate against your SLO (Service Level Objectives)
+
+KLT sits in the job scheduler and can trace the deployment
+from start to end.
+KLT is also application aware,
+so we can extend the deployment
+with tasks and evaluations that
+are run either before or after the actual deployment.
+You can also validate any metric,
+either pre- or post-deployment,
+using the metrics from the Keptn Metrics Server introduced in
+[Getting started with Keptn metrics](../metrics).
+This means that you can be sure that the environment is healthy
+and has adequate resources before you begin the deployment.
+After the deployment succeeds,
+use Keptn metrics to confirm that your deployed software is really healthy --
+not just that the pods are running but validate against SLOs
+such as performance and user experience.
+You can also check for new logs that came in from a log monitoring solution.
+
+## Using this exercise
+
+This exercise is based on the
+[simplenode-dev](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd)
+example.
+You can clone that repo to access it locally
+or just look at it for examples
+as you implement the functionality "from scratch"
+on your local Kubernetes deployment cluster.
+
+The steps to implement pre- and post-deployment orchestration are:
+
+1. [Bring or create a Kubernetes cluster](#bring-or-create-a-kubernetes-deployment-cluster)
+1. [Install the Keptn Lifecycle Toolkit on your cluster](#install-klt-on-your-cluster)
+1. [Enable KLT for your cluster](#enable-klt-for-your-cluster)
+1. [Integrate KLT with your cluster](#integrate-klt-with-your-cluster)
+1. [Define evaluations to be performed pre- and post-deployment](#define-evaluations-to-be-performed-pre--and-post-deployment)
+1. [Define tasks to be performed pre- and post-deployment](#define-tasks-to-be-performed-pre--and-post-deployment)
```

Comment:
In the Metric getting started we pointed readers to "Installation" subsections directly, so we don't replicate text 
Namely the following:
```suggestion
1. [Bring or create a Kubernetes cluster](../../install/k8s/)
1. [Install the Keptn Lifecycle Toolkit and enable it](../../install/install/)
1. [Integrate KLT with your cluster](../../implementing/integrate/)
1. [Define evaluations to be performed pre- and post-deployment](#define-evaluations-to-be-performed-pre--and-post-deployment)
1. [Define tasks to be performed pre- and post-deployment](#define-tasks-to-be-performed-pre--and-post-deployment)
```